simple_rl/tasks/navigation/NavigationMDP.py

```diff
@@ -2,6 +2,7 @@
 
 # Python imports.
 from __future__ import print_function
+import copy
 import numpy as np
 from collections import defaultdict
 from simple_rl.tasks import GridWorldMDP
```

Comment:
I tend to separate out the simple_rl imports, so:

```
# Python imports.
import copy
import numpy as np

# Other imports.
import simple_rl
```
Of course not a huge change -- just something to be aware about for standardizing with future commits.

Reply:
Not a problem, thanks!